README-EN.md

```diff
@@ -1,64 +1,103 @@
 Visit the [Publication Guide](el-bid.github.io/guia-de-publicacion/) (currently available in Spanish) for more information about how to publish digital tools.
```

Comment:
Oh, also, I wasn't sure about this but ended up deleting it from the other IDB repo. I think it's part of the template.

Reply:
Yeah, I wasn't sure either. It didn't seem like guidance for the filling out the README, so I thought it was meant to stay